src/ethereum/rfc003/erc20_htlc.rs

```diff
@@ -53,7 +53,7 @@ impl Erc20Htlc {
     pub fn refund_tx_gas_limit() -> u64 {
         // 20_705 consumed in local test for successful refunding
```

Comment:
If more gas is needed, where did these numbers come from? 😄

Reply:
You are asking an excellent question and I simply do not have the answer to this question.

I simply do not understand the behaviour here and would like to get my head out of this PR to later investigate more.

When running the blockchain-contract test, no redeem cost more than 35000. However, if I were to set the redeem gas limit to 50000, I would get an "out of gas" error.

How come I need to set the gas limit to 50000 when I reproducibly know that this transaction will not consume more than 35000 gas???

In any case, I am guessing something similar is happening on the refund side.

Reply:
@da-kami : the initial values were taken from manual deployment and checking the used gas. 
I have to be honest: I did not update this function when I updated the contracts and  it seems like this was not used in the tests.